src/commands/user/profile.ts

```diff
@@ -77,14 +78,23 @@ class ProfileCommand extends Command {
 				: 'Single';
 		}
 
+		let userTime: string = 'No timezone set.';
+
+		if (userModel.timezone || userModel.timezone === 0) {
```

Comment:
I think a `userModel.timezone !== null` or `typeof userModel.timezone === 'number'` would be more elegant.